apps/web/src/server/lib/dates.js

```diff
@@ -30,3 +36,24 @@ export const isDateInstance = (date) => {
 		!Number.isNaN(date.getFullYear())
 	);
 };
+
+/**
+ * @param {Date | number | string | null | undefined} date
+ * @param {{ condensed?: boolean }} options
+ * @returns {string}
+ */
+export function dateToDisplayDate(date, { condensed = false } = {}) {
+	if (typeof date === 'undefined' || date === null) {
+		return '';
+	}
```

Comment:
I would also add a check for '0000-00-00' date which can crop up and we don't want to show/format it, as annoyingly it would pass validation from date-fns isValid(). 
Eg we do it in `apps/web/src/server/applications/case/representations/utils/is-relevant-reps-period-closed.js` and `forms-web-app/src/pages/projects/_utils/get-application-data.js`

Reply:
Maybe it's worth having that check in `apps/web/src/server/applications/case/representations/utils/get-key-dates.js` as well, so it would fail validation before it even gets to formatting, and have the formatting change as a failsafe